app/cli/internal/action/workflow_run_list.go

```diff
@@ -110,14 +110,17 @@ func pbWorkflowRunItemToAction(in *pb.WorkflowRunItem) *WorkflowRunItem {
 }
 
 func humanizedRunnerType(in v1.CraftingSchema_Runner_RunnerType) string {
-	switch in {
-	case *v1.CraftingSchema_Runner_GITHUB_ACTION.Enum():
-		return "GitHub"
-	case *v1.CraftingSchema_Runner_GITLAB_PIPELINE.Enum():
-		return "GitLab"
-	case *v1.CraftingSchema_Runner_AZURE_PIPELINE.Enum():
-		return "Azure Pipeline"
-	default:
+	mapping := map[v1.CraftingSchema_Runner_RunnerType]string{
```

Comment:
This code (and the previous one) has a flaw.

It treats the same the fact that a runner is not specified (generic) or it's not known by the CLI because it's a new type (your Jenkins case)

I would update this code to handle both cases, which means 

* adding to the mapping `CraftingSchema_Runner_RUNNER_TYPE_UNSPECIFIED: "Unspecified"`
* the returned value for things that are not in the map should be called something 
 like "Unknown" 


What do you think?

Reply:
I noticed that. I didn't do anything because I thought it was intentional for some reason.
I agree wityh your solution. I'll do that.
Thanks.